apps/omg_child_chain/test/omg_child_chain/fees/file_adapter_test.exs

```diff
@@ -40,13 +40,7 @@ defmodule OMG.ChildChain.FileAdapterTest do
     }
   }
 
-  setup do
-    old_value = Application.get_env(:omg_child_chain, :fee_specs_file_name)
-
-    on_exit(fn ->
-      :ok = Application.put_env(:omg_child_chain, :fee_specs_file_name, old_value)
-    end)
-  end
+  @ignored_actual_fees %{}
```

Comment:
current?

Reply:
came up to different name, but it's a good point